skopt/gp_opt.py

```diff
@@ -94,19 +94,17 @@ def gp_minimize(func, bounds=None, search="sampling", random_state=None,
 
     Returns
     -------
-    x_val: array-like
-        Parameter value corresponding to the best function value.
-
-    func_val: float
-        Function minimum.
-
-    d: dict
-        d["models"]: List of GP models of len maxiter. d["models"][i]
-        corresponds to the GP model fit in iteration i.
-        d["x_iters"]: d["x_iters"][i] corresponds to the "best" x in
-        iteration i.
-        d["func_vals]: d["func_vals"][i] corresponds to the minimum
-        function value in iteration i.
+    res: OptimizeResult
+        The optimization result returned as a OptimizeResult object.
+        Important attributes are
+        ``x`` - float, the optimization solution,
+        ``fun`` - float, the value of the function at the optimum,
+        ``models``- gp_models[i]. the prior on the function fit at
+                       iteration[i].
+        ``func_vals`` - the minimum function value attained at the ith
+                        iteration.
+        ``x_iters`` - the value of ``x`` corresponding to the minimum
+                      function value at the ith iteration.
```

Comment:
I would add a note pointing to https://github.com/scipy/scipy/blob/master/scipy/optimize/optimize.py#L75-L130